common/extensions/api/brave_rewards.json

```diff
@@ -1201,6 +1201,47 @@
           }
         ]
       },
+      {
+        "name": "getScheduledCaptchaInfo",
+        "type": "function",
+        "description": "Gets all info required to retrieve a scheduled captcha (if any) for the user.",
+        "parameters": [
+          {
+            "type": "function",
+            "name": "callback",
+            "parameters": [
+              {
+                "name": "result",
+                "type": "boolean"
+              },
+              {
+                "name": "properties",
+                "type": "object",
+                "optional": true,
+                "properties": {
+                  "url": {
+                    "type": "string"
+                  },
+                  "maxAttemptsExceeded": {
+                    "type": "boolean"
+                  }
+                }
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "name": "updateScheduledCaptchaResult",
+        "type": "function",
+        "description": "",
```

Comment:
Marking this line in case you just forgot to add a description.

Reply:
Yes, added.